src/Spryker/Client/RabbitMq/Model/Connection/ConnectionManager.php

```diff
@@ -154,4 +158,14 @@ protected function getDefaultQueueConnectionTransfer(): QueueConnectionTransfer
 
         throw new DefaultConnectionNotFoundException(static::EXCEPTION_MESSAGE_DEFAULT_CONNECTION_NOT_FOUND);
     }
+
+    /**
+     * @return \Spryker\Client\RabbitMq\Model\Connection\ConnectionInterface
+     */
+    protected function getDefaultConnection(): ConnectionInterface
```

Comment:
Mb we can just make it public and return a connection? In this case I guess we can use it as a return value in the new client method

Reply:
changed